Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -1111,6 +1110,19 @@ void AODProducerWorkflowDPL::fillMCTrackLabelsTable(const MCTrackLabelCursorType
       }
     }
   }
+
+  // filling the tables with the strangeness tracking labels
+  auto sTrackLabels = data.getStrangeTracksMCLabels();
+  for (auto& collStrTrk : mCollisionStrTrk) {
+    if (mVtxToTableCollID.find(collStrTrk.first) == mVtxToTableCollID.end() || mVtxToTableCollID[collStrTrk.first] != vertexId) {
```

Comment:
same question about excessive mVtxToTableCollID check. Additionally, the check is still done even for the "orphan" vertexID=-1, though by construction there are no strange tracks in this case, right?